.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug Report
+about: Report a bug encountered while operating on CStor Volumes
+labels: kind/bug
+
+---
+
+<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
+-->
+
+**What happened:**
+
+**What you expected to happen**:
+
+**The output of the following commands will help us better understand what's going on**:
+(Pasting long output into a [GitHub gist](https://gist.github.com) or other pastebin is fine)
+
+* `kubectl get cstorvolume <pv-name> -n <openebs_namespace> -o yaml`
```

Comment:
CVR and CSP also is probably good to get.

Reply:
Updated with corresponding comments